lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb

```diff
@@ -441,26 +453,38 @@ def disallows_lower_value
         end
 
         def allows_minimum_value
+          return true unless @minimum
+
```

Comment:
I'd say we could change this by removing the guard clause to stay consistent with the other methods in this class.

```suggestion
          @minimum.nil? || allows_value_of(@minimum, @low_message)
```

Reply:
That makes a lot of sense, thanks! I've updated the nil checks. 👍